R/input-checks.R

```diff
@@ -0,0 +1,53 @@
+
+#' Find missing input pieces.
+#'
+#' find_tests_wo_reqs()` returns IDs from the test results that are not
+#' associated with a requirement. `find_reqs_wo_tests()` goes in the reverse
+#' direction, returning requirements with test IDs that aren't found in the test
+#' results. `find_reqs_wo_stories()` returns requirements that are not linked to
+#' a story.
+#'
+#' @param merged_inputs Tibble with stories, requirements and tests, as returned
+#'   by [create_validation_docs()].
+#' @return Tibble with the missing items.
+#' @seealso [create_validation_docs()], [input_formats]
+#' @importFrom dplyr arrange filter select
+#' @importFrom rlang .data
+#' @importFrom tidyr unnest
+#' @export
+find_tests_wo_reqs <- function(merged_inputs) {
```

Comment:
For some reason I was on the fence about "wo" or "without" in these function names, but looking at this again, I think it should be spelled out.